tests/ci/codebuild/release/al2_aarch64_test.yml

```diff
@@ -0,0 +1,62 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+version: 0.2
+
+env:
+  variables:
+    GO_TAG: go1.18.3
+phases:
+  # TODO: Everything in the install section should be moved into a prebuilt image
+  install:
+    runtime-versions:
+      java: corretto11
+    commands:
+      - yum -y update
+      - yum -y group install 'Development Tools'
+      # - curl -L -O https://github.com/Kitware/CMake/releases/download/v3.24.0-rc2/cmake-3.24.0-rc2.tar.gz
+      - curl -L -O https://downloads.sourceforge.net/ltp/lcov-1.14-1.noarch.rpm
+      # - tar xzf cmake-3.24.0-rc2.tar.gz
+      # - (cd cmake-3.24.0-rc2 && ./bootstrap && make && make install)
+      - yum -y install cmake3
+      - yum -y install gsl-devel wget
+      - mkdir go_tmp
+      - cd go_tmp
+      - wget https://dl.google.com/go/$GO_TAG.linux-arm64.tar.gz
+      - tar -xvf $GO_TAG.linux-arm64.tar.gz
+      - mv go ../go_root
+      - cd ..
+  build:
+    commands:
+      # TODO: Move this to the docker image also
+      - export GOROOT=$(pwd)/go_root
+      - export GO111MODULE=on
+      - export PATH=$GOROOT/bin:$PATH
+      # TODO: Can we move the recursive pull logic upstream?
+      - git submodule update --init --recursive
+      - pwd
+      - ls
+      # TODO: Pull from maven rather than from local location
+      # TODO: Do we want to reintroduce dieharder for some configurations?
+      # - ./gradlew -DFIPS=${FIPS} -DstagingUrl=${STAGING_URL} test test_extra_checks test_integration test_integration_extra_checks
+      - ./gradlew -DcmakeBin=cmake3 -DFIPS=${FIPS} -DstagingProperties=true test test_extra_checks test_integration test_integration_extra_checks
+      - ./gradlew -DcmakeBin=cmake3 -DFIPS=${FIPS} -DTEST_JAVA_HOME=$JAVA_8_HOME -DTEST_JAVA_MAJOR_VERSION=8 -DstagingProperties=true minimal_clean test test_extra_checks test_integration test_integration_extra_checks
+artifacts:
+  files:
+    - 'lib/**/*'
+  discard-paths: no
+  base-directory: 'build'
+  name: $(date --iso-8601=s)
+reports:
+  unit-tests:
+    files:
+      - 'build/reports/unit-tests/**'
+    #base-directory: location
+    discard-paths: yes
+    file-format: JunitXml
```

Comment:
nice! where can we view these reports?

Reply:
whoah! i didn't know codebuild understood junit XML. this is pretty neat.

![Screen Shot 2022-07-13 at 2 39 39 PM](https://user-images.githubusercontent.com/3589880/178807138-6b529dd7-f67d-40ae-85c5-7f2804ce2ac1.png)

Reply:
This is something we should probably port to our standard CI jobs. Once this change is merged in all you need to do is update the build-specs. If you want, I can do that now too.